아래 코드는 R에서 작성되었습니다.      
런타임 유형을 변경해서 실행하여야 합니다.

https://colab.research.google.com/notebook#create=true&language=r

In [12]:
package_version(R.version)

[1] ‘4.3.0’

In [24]:
#어절 길이 비교
system("gdown --id 1ECoINUqQCjGKFWuVy3_CVakrymO278VM")
data = read.csv("Generated_word_length0522.csv")


In [10]:
head(data)

Source_word_length Generated_word_length
1 4                  4                    
2 6                  4                    
3 3                  2                    
4 1                  3                    
5 4                  3                    
6 4                  2

In [11]:
# t-검정 수행
result <- t.test(data$Source_word_length, data$Generated_word_length)

# 결과 출력
print(result)


	Welch Two Sample t-test

data:  data$Source_word_length and data$Generated_word_length
t = 1.3392, df = 922.68, p-value = 0.1808
alternative hypothesis: true difference in means is not equal to 0
95 percent confidence interval:
 -0.09773452  0.51773452
sample estimates:
mean of x mean of y 
    4.068     3.858 



어절 길이에 대한 통계적 유의성을 확인하기 위해서는 통계 분석 도구인 R(v.4.3.0)을 사용했다.	 Welch Two Sample t-검정 수행 결과는 P 값은 1.34로 유의 수준 0.05보다 높으므로 두 표본의 평균 사이에는 통계적으로 유의미한 차이가 있다고 할 수 없다. 같은 방법으로 음절 길이에 대해서 통계적 치이가 있는지 검토하였다.

In [25]:
#음절 길이 비교
system("gdown --id 1FTDrnCpGkJydRjStTLMyjsBJ1NqDwUdZ")
data2 = read.csv("Source_syllable_length0522.csv")

In [19]:
head(data2)

Source_syllable_length Generated_syllable_length
1 13                     13                       
2 18                     12                       
3  9                      8                       
4  6                     11                       
5 14                     12                       
6 12                     10

In [23]:
# t-검정 수행
result <- t.test(data2$Source_syllable_length, data2$Generated_syllable_length)

# 결과 출력
print(result)


	Welch Two Sample t-test

data:  data2$Source_syllable_length and data2$Generated_syllable_length
t = 0.06743, df = 935.06, p-value = 0.9463
alternative hypothesis: true difference in means is not equal to 0
95 percent confidence interval:
 -1.067963  1.143963
sample estimates:
mean of x mean of y 
   14.620    14.582 



음절 길이에 대한 Welch Two Sample 결과 P 값은 약 0.95로 유의 수준(p-값, 0.05)보다 높으므로 두 표본의 평균 사이에는 통계적으로 유의미한 차이가 있다고 할 수 없다.